src/servers/trigger.ts

```diff
@@ -166,19 +164,18 @@ export const onInvoiceUpdate = async (invoice) => {
 }
 
 const publishCurrentPrice = async () => {
-  const satUsdPrice = await getCurrentPrice()
-  if (satUsdPrice) {
-    pubsub.publish(SAT_USDCENT_PRICE, { satUsdCentPrice: 100 * satUsdPrice })
-  }
+  const satsPrice = await getCurrentPrice()
+  if (satsPrice instanceof Error) return
+
+  pubsub.publish(SAT_USDCENT_PRICE, { satUsdCentPrice: 100 * satsPrice })
 }
 
 const updatePriceForChart = () => {
-  const price = new Price({ logger: baseLogger })
   const interval = 1000 * 30
   return setInterval(async function () {
     try {
-      await price.update()
-      publishCurrentPrice()
+      await updatePriceHistory()
+      await publishCurrentPrice()
```

Comment:
Just for my knowledge, why did we change to this to explicit `await`?

Reply:
this is in trigger so we dont need fire and forget, also this makes the last price publish consistent specially for galoy-pay